### DISCUSSION TITLE
Feature Request:Add  a circular button menu

### DISCUSSION THREAD

**Discussion by Kos-M** [saadeghi/daisyui#2783] [Ideas / feature requests]
It would be awesome , if somehow we had native such a component.

Example lib that founf the idea : [react-planet](https://www.npmjs.com/package/react-planet)

**Comment by NikarashiHatsu** [comment]
When you want to add interactivity, there's a lot of JavaScript libraries out there that provide the functionality. But put it into mind that this is a CSS library, not a JS library (though technically it is JS).

**Comment by Kos-M** [comment]
Valid point indeed. Im not sure if this can be done with only css thought..
And let the user to add actions logic.

But in the spirit of offering to devs a variety of components that can choose to add when creating UIs its worth the trouble , i think :)

For me at least , its a factor when deciding which lib/framework to choose for my next projects.
More componets the better. ...
Besides the look and feel of course..
